news/2021-03-18-community-contributors-2020.md

```diff
@@ -0,0 +1,246 @@
+
+His unrelenting drive to push full length beatmaps has never faltered, and he remains one of the few nominators to make grand efforts to push maps from as many unique mappers as possible.
+
+**Mordred** is becoming synonymous with the Elite title, and for very, very good reason.
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/lasse.jpg)](https://osu.ppy.sh/users/896613)
+
+If being mentioned in a Community Contributor article twice is not prize enough, **Lasse**'s outstanding performance as a Beatmap Nominator (while also being a member of the NAT) should truly emphasise just how incredible his work ethic is. 
+
+His presence in the modding scene is difficult to ignore as he consistently checks and pushes beatmaps with superb accuracy and speed, tending to over 100 unique mappers in a single year.
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/agatsu.jpg)](https://osu.ppy.sh/users/5579871)
+
+Remaining one of the most active beatmap nominators to date, **Agatsu** is renowned for the level of care he puts into each review and is well respected for being an indisputably solid and supportive modder.
+
+He has since risen to take his place among the Nomination Assessment Team, if that were not indicative enough of his considerable expertise and drive to see the modding and mapping scene improve.
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/baar_vendel.jpg)](https://osu.ppy.sh/users/8679346)
+
+Despite only entering the fray less than a year ago, **BaAR_Vendel** has proven themselves to be utterly exceptional at their role.
+
+They have tirelessly helped numerous mappers earn their first ranked map and assisted multitudes more in getting the fabled second set of eyes needed to give a map that extra push through the process, earning themselves a strong reputation as a selfless individual who deeply cares about the mapping and modding community at large.
+
+Very befitting of an Elite Nominator, in our eyes.
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/milan.jpg)](https://osu.ppy.sh/users/1052994)
+
+While perhaps not the most active name on the list so far, **Milan-** is certainly one of the most consistent. They have focused on helping as many unique mappers as possible with special attention given to those new to the scene, all without compromising on the quality of the maps that they push forward into Ranked - especially notable as they've received nearly no disqualifications despite a *very* packed 2020.
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/eiri.jpg)](https://osu.ppy.sh/users/3388410)
+
+Living in two worlds, **eiri-** is perhaps one of the most interconnected hybrid beatmap nominators there has ever been. Proficient in not one but *two* game modes, **eiri-** has pulling twice their weight in pushing over 80 osu!taiko beatmaps in 2020 alone, not to mention the fact that they were an Elite Nominator recipient last year as well... for osu!.
```

Comment:
weird to go on about how they're a proficient osu+taiko nominator, then only list the # of maps they pushed for taiko. would include the osu stat too

Reply:
updated wording for this